src/paginate.ts

```diff
@@ -21,19 +21,12 @@ export async function paginate<T>(
     ...(searchOptions as object),
   });
 
+  const isNext = route && (total / limit >= (page + 1));
```

Comment:
Do we need these constants defined?

Reply:
the constants are not required but it is a good practice to do the code more readable for the humans ☺️. I think that the code will be clearer using these constants.

however, if you doesnt define the constants, I believe that the refactoring that improve the code is the avoid of if-logic.

Reply:
I personally don't but I'll approve as soon as I can :)